new_core_concepts/modules/ROOT/pages/typeql/entities-relations-attributes.adoc

```diff
@@ -1,4 +1,137 @@
 = Entities, Relations, Attributes
+// TODO: Out links to manual / fundamentals articles
 
-[placeholder]
-TypeQL data model concepts. 
\ No newline at end of file
+TypeDB implements the Polymorphic Entity-Relation-Attribute (PERA) data model.
+A TypeDB schema defines a hierarchy of these types, and how they interact with each other through interface-types.
+
+=== Entity
+
+Entities are standalone types, which exist independent of other types.
+
+E.g. A `person` exists, regardless of whether they are involved in friendships or have a name.
+[,typeql]
+----
+entity person;
+----
+
+=== Relation
+Relations are types with one-or-more associated "role" types.
+An instance of a relation "depends" on instances of other types which "play" these roles.
+
+E.g. An `employment` relation exists to relate an `employer` to an `employee`.
+An `employment` instance cannot exist without the other instances which play these roles.
```

Comment:
I think it's often useful to repeat an important and possibly confusing concept a couple times:

```
Relations are types with one-or-more associated "role" types, <which you define in the schema>.

An instance of a relation "depends" on instances of other types which "play" these roles <you define>. <This is the primary distinction between entities, and relations!>


E.g. An `employment` relation exists to relate an `employer` to an `employee`.

<Consider that an> `employment` <relation> instance cannot exist without the other instances which play these roles<, hence the idea that `employment`s "depend" on the participating players.>
```